src/Carousel/widget/components/CarouselCaption.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Original code can be found at:
+ * https://github.com/react-bootstrap/react-bootstrap/blob/master/src/CarouselCaption.js
+ * 
+ * This code is under the MIT license found here:
+ * https://github.com/react-bootstrap/react-bootstrap/blob/master/LICENSE
+ */
+
+import classNames = require("Carousel/lib/classnames");
+import * as React from "Carousel/lib/react";
+
+import { prefix } from "../utils/bootstrapUtils";
+
+interface CaptionProps extends React.Props<CarouselCaption> {
+    componentClass?: string;
+    className?: string;
+    bootstrapClass?: string;
+    elementProps?: {};
+}
+
+class CarouselCaption extends React.Component<CaptionProps, {}> {
```

Comment:
This is a perfect candidate for a stateless function, which React also recommends: https://facebook.github.io/react/docs/reusable-components.html#stateless-functions.